File: cs102/web-7/lab4.c
#include <stdio.h>
int main (int argc, char** argv) {
    fprintf(stdout,"Content-type: text/html\n\n");
    fprintf(stdout,"<TITLE>Lab 4</TITLE>\n");
    fprintf(stdout,"<h1>Lab 4</h1>\n");
    fprintf(stdout,"<html>\n");
    fprintf(stdout,"<body>\n");
    fprintf(stdout,"<table>\n");
    for (int x = 0; x<100; x=x+1) {
        fprintf(stdout,"<tr><td>%d</td><td>%d</td></tr>\n", x+1, (x+1)*100);
    }
    fprintf(stdout,"</table>\n");
    fprintf(stdout,"</html>\n");
    fprintf(stdout,"</body>\n");
    return 0;
}  

File: cs102/web-7/lab4.py
#!/usr/bin/python
print( "Content-type: text/html\n\n" )
print("<TITLE>Lab 4</TITLE>")
print("<h1>Lab 4</h1>")
print( """
        <html>
        <body>
	<table>
""" )
for x in range(100):
    print("<tr><td>%s</td><td>%s</td></tr>" % (x+1, (x+1)*100))
print( """
        </table>
	</body>
        </html>
""" )
